src/frontend/src/lib/components/ui/DropdownButton.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+	import { IconExpandMore } from '@dfinity/gix-components';
+
+	export let disabled = false;
+	export let button: HTMLButtonElement | undefined = undefined;
+	export let ariaLabel: string;
+	export let opened = false;
+</script>
+
+<button
+	class="text-secondary min-w-72 justify-between gap-2 rounded-xl border border-light-grey bg-white px-4 py-3 text-left font-medium leading-5 text-inherit hover:border-blue-ribbon"
+	bind:this={button}
+	on:click
+	aria-label={ariaLabel}
+	{disabled}
+	class:opacity-50={disabled}
+	class:border={opened}
```

Comment:
You need this, the button has already a border in the class?

Reply:
right! removed